packages/core/src/App/Constants/routes-config.js

```diff
@@ -52,7 +52,7 @@ const getModules = ({ is_deriv_crypto }) => {
         {
             path: routes.account,
             component: Account,
-            getTitle: () => localize('Accounts Settings'),
+            getTitle: () => localize('Account Settings'),
```

Comment:
If this change is intentional, please label the relative trello card as "needs translation".

Reply:
@reza-fs  Different titles for this section at [here](https://github.com/binary-com/deriv-app/blob/a9237bb2434b7f7ebd5d378f10f990de6e589d9e/packages/account/src/Constants/routes-config.js#L37) and [here](https://github.com/binary-com/deriv-app/blob/a9237bb2434b7f7ebd5d378f10f990de6e589d9e/packages/core/src/App/Constants/routes-config.js#L55) causes a sudden content change in the login prompt so I just unified them to `Account Settings`. As this text existed [here](https://github.com/binary-com/deriv-app/blob/a9237bb2434b7f7ebd5d378f10f990de6e589d9e/packages/account/src/Constants/routes-config.js#L37) do I need to add a "needs translation" label to the card?

Reply:
@mustofa-binary Do we need a translation for this case?

Reply:
Ah, that clears! I guess we won't need translation on this since we already have one for it.